src/main/java/calculator/repository/MemorySeparatorRepository.java

```diff
@@ -0,0 +1,19 @@
+package calculator.repository;
+
+import calculator.model.Separator;
+import java.util.HashSet;
+import java.util.Set;
+
+public class MemorySeparatorRepository implements SeparatorRepository{
+    private static final Set<Separator> store = new HashSet<>();
+
+    @Override
+    public void save(Separator separator) {
+        store.add(separator);
+    }
+
+    @Override
+    public Set<Separator> findAll() {
+        return store;
```

Comment:
자료형을 그대로 넘겨버리면 주소를 공유하게 되어 외부에서 값이 변경될 위험이 있습니다.
Spring Data JPA와 마찬가지로 호출할 때마다 새로운 자료형을 생성해서 넘겨주는 것이 좋을 것 같습니다
```suggestion
        return new HashSet<>(store);
```